helix/principles/file-structure/modules.rst

```diff
@@ -1,15 +1,16 @@
 Module structure
 ~~~~~~~~~~~~~~~~
 
-Each module has the following structure:
+Each module has the following structure, note that not all modules will contain all of the elements:
```

Comment:
Might be worthwhile explaining why a module wouldn't contain all elements, i.e. a module may contain code logic that impacts the Commerce Engine, however no part of the module needs to be propagated to the Website.